libs/SerialNumberKit/include/SerialNumberKit.h

```diff
@@ -5,21 +5,28 @@
 #pragma once
 
 #include <array>
+#include <filesystem>
 #include <interface/drivers/MCU.h>
+#include <span>
 
 namespace leka {
 
 class SerialNumberKit
 {
-	static constexpr size_t SN_SIZE		  = 33;
-	static constexpr size_t SHORT_SN_SIZE = 16 + 1;
+	static constexpr size_t SN_SIZE			= 31 + 1;
+	static constexpr size_t SHORT_SN_SIZE	= 15 + 1;
```

Comment:
Ok for `std::size_t`

I let your suggestion to explain `31+1` instead of `32`.  
In fact it depends on the use of `serial_number`, since bluetooth advertising name and S/N device information characteristic both set S/N via `const char *`, the `serial_number` must implement a null terminated character

Reply:
Use `ShortSN + '\0'` isntead of `SSN + '\0``